### PR TITLE
[16.0][MIG] delivery_package_fee: Migrate to version 16.0

### PR DESCRIPTION
**Pull request by tuantrantg**
Note for the reviewer:
- The `price` field (of `product.product` model) in 14.0 is replaced by the `_get_contextual_price` function (of model `product.product`) in 16.0. You can see the details in [here](https://github.com/odoo/odoo/commit/9e99a9df464d97a74ca320d200599f7dce2d3050)

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

File: delivery_package_fee/tests/test_package_fee.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def _create_sale(cls):
     def _add_sale_carrier(cls, sale, carrier):
         delivery_wizard = Form(
             cls.env["choose.delivery.carrier"].with_context(
-                {"default_order_id": sale.id, "default_carrier_id": carrier.id}
+                **{"default_order_id": sale.id, "default_carrier_id": carrier.id}
```
Comment (rousseldenis) [inline delivery_package_fee/tests/test_package_fee.py]: You can avoid such notation setting parameters directly:

`(...).with_context(default_order_id=sale.id, ...)`

**Review by rousseldenis — APPROVED** [review]
Code review. Minor change

**Review by santostelmo — APPROVED** [review]

**Comment by yvaucher** [comment]
@tuantrantg Please see @rousseldenis comment

**Comment by tuantrantg** [comment]
> @tuantrantg Please see @rousseldenis comment

Hi @rousseldenis , the PR updated, please review it again. Thank you

**Pull request merged**